stacks/product-jenkins/variables.tf

```diff
@@ -16,6 +16,10 @@ variable "config_context" {
   default = null
 }
 
+variable "config_path" {
+  default = "~/.kube/config"
```

Comment:
I haven't tested this, but I have a feeling that this is going to break our usage of the jenkins product from within the cluster, since the product operator isn't using a kubeconfig to authenticate.  we should be really careful with the defaults we set here.